src/rgw/rgw_lua_background.cc

```diff
@@ -0,0 +1,62 @@
+  #include "rgw_lua_background.h"
+  #include "rgw_lua.h"
+  #include "rgw_lua_utils.h"
+  #include "include/ceph_assert.h"
+  #include <lua.hpp>
+
+  namespace rgw::lua {
+
+  void Background::shutdown(){
+    this->stop();
+    runner.join();
+  }
+  void Background::stop(){
+    stopped = true;
+  }
+
+  //(1) Loads the script from the object
+  //(2) Executes the script
+  //(3) Sleep (configurable)
+  void Background::run() {
+
+    create_metatable<rgw::lua::RGWTable>(L, true, &rgw_map_s);
```

Comment:
probably better to expose a `create_background_metatable()` interface of the `Background` class
and make `rgw_map_s` private